pkg/madmin/info-commands.go

```diff
@@ -283,9 +283,9 @@ type Disk struct {
 	State           string  `json:"state,omitempty"`
 	UUID            string  `json:"uuid,omitempty"`
 	Model           string  `json:"model,omitempty"`
-	TotalSpace      uint64  `json:"totalspace,omitempty"`
-	UsedSpace       uint64  `json:"usedspace,omitempty"`
-	AvailableSpace  uint64  `json:"availspace,omitempty"`
+	TotalSpace      float64 `json:"totalspace,omitempty"`
+	UsedSpace       float64 `json:"usedspace,omitempty"`
+	AvailableSpace  float64 `json:"availspace,omitempty"`
```

Comment:
No. We should report exact numbers and leave it to the client to round/format.

Reply:
The issue we have is the server returns bigger JSON output and the goal is to reduce the size. Doing at client side is meaningless here.